packages/docs/src/styles/_examples.scss

```diff
@@ -213,6 +213,12 @@ $options-margin: $pt-grid-size * 3;
   animation: docs-wiggle-rotate $pt-transition-duration $pt-transition-ease infinite;
 }
 
+.docs-loading-skeleton-example-box {
+  flex: 1;
+  margin-right: $pt-grid-size * 1.5;
+  height: $pt-grid-size * 15;
```

Comment:
Hmm, so is this how any skeleton container should be used? What's wrong with the container size changing? Seems fine to me, in a real app it will only adjust once since the skeleton is only shown initially